x-pack/test_serverless/api_integration/services/slo_api.ts

```diff
@@ -87,6 +66,19 @@ export function SloApiProvider({ getService }: FtrProviderContext) {
       return body;
     },
 
+    async update(
+      { sloId, slo }: { sloId: string; slo: UpdateSLOInput },
+      roleAuthc: RoleCredentials
+    ) {
+      const { body } = await supertest
```

Comment:
I just noticed that `SloApiProvider` is using `supertest` service. This service is authenticated with operator user and will use it to run API calls (ignoring `roleAuthc.apiKeyHeader`). It is not a right approach, please update the service to `supertestWithoutAuth`. This one has no authentication cached and will use provided API key / Cookie header to make API calls.

Reply:
Thanks, will do

Reply:
resolved in https://github.com/elastic/kibana/pull/195316/commits/d23a04eaca33994030822dc8ab8f534f37e0b6c3

Reply:
@kdelemme We are actively working on migrating our current tests to the deployment agnostic framework. slo_api service in serverless folder will soon be removed. The [slo_api](https://github.com/elastic/kibana/blob/main/x-pack/test/api_integration/deployment_agnostic/services/slo_api.ts) service has already been migrated to the deployment agnostic folder and @dominiqueclarke is working into [migrating the slo tests](https://github.com/elastic/kibana/issues/183397) to the new framework.